llvm-build/Microsoft.Quantum.LlvmBindings.Native.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Quantum.LlvmBindings.Native</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>QuantumEngineering, Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>LLVM native support for LLVM Bindings.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <file src=".\drops\libLLVM.dll" target="runtimes\win-x64\native\libLLVM.dll" />
+    <file src=".\drops\libLLVM.dylib" target="runtimes\osx-x64\native\libLLVM.dylib" />
+    <file src=".\drops\libLLVM.so" target="runtimes\linux-x64\native\libLLVM.so" />
```

Comment:
For other dynamic libraries the Win version typically does not have the `lib` prefix (in `libLLVM.dll`).

Reply:
A notable exception would be `libomp` ;). The naming here is intentional, and matches what was done by LLVMSharp in their interop package (see for example https://github.com/dotnet/LLVMSharp/blob/ad498be4815dc4699981ba61c9ade30a740d5904/sources/LLVMSharp.Interop/Extensions/LLVM.Manual.cs#L12 which explicitly uses `"libLLVM"` for all platforms).